source/codegen/metadata/nidcpower/CHANGES.md

```diff
@@ -16,7 +16,7 @@ The following function name was capitalised:
 The following functions were tagged with 'init_method': True, to ensure their generated service handler registers the new session with the session_repository:
 - InitializeWithIndependentChannels
 - InitializeWithChannels
-- InitExtCal
+- InitExtCal : Added a 'custom_close' tag to this function, since this API has a corresponding close function called 'CloseExtCal'
```

Comment:
I think we should change wording to something like this to record both changes without causing confusion -> addition of `init_method` tag and addition of custom_close tag.
____________________
- InitExtCal
  - Also added a 'custom_close' tag to this function, since this API has a corresponding close function called 'CloseExtCal'

Reply:
Yes, this is more indicative. I have made this change.